tests/test_senv.py

```diff
@@ -3,32 +3,48 @@
 from streaming_data_types import SERIALISERS, DESERIALISERS
 from datetime import datetime
 from streaming_data_types.fbschemas.sample_environment_senv.Location import Location
+import pytest
 
 
-class TestSerialisationSenv:
-    original_entry = {
-        "name": "some_name",
-        "timestamp": datetime.now(),
-        "channel": 42,
-        "message_counter": 123456,
-        "sample_ts_delta": 0.005,
-        "values": np.arange(100),
-        "value_timestamps": None,
-        "ts_location": Location.End
-    }
+entry_1 = {
+    "name": "some_name",
+    "timestamp": datetime.now(),
+    "channel": 42,
+    "message_counter": 123456,
+    "sample_ts_delta": 0.005,
+    "values": np.arange(100, dtype=np.uint16),
+    "value_timestamps": np.arange(50) + 1111,
+    "ts_location": Location.End
+}
+
+entry_2 = {
+    "name": "some_name_other_name",
+    "timestamp": datetime.now(),
+    "channel": 11,
+    "message_counter": 654321,
+    "sample_ts_delta": 1.666,
+    "values": np.arange(1000, dtype=np.int64),
+    "value_timestamps": None,
+    "ts_location": Location.Middle
+}
 
-    def test_serialises_and_deserialises_senv(self):
-        buf = serialise_senv(**self.original_entry)
+
+class TestSerialisationSenv:
+    @pytest.mark.parametrize("input_entry", [entry_1, entry_2])
```

Comment:
Minor point: when mutability is not needed, it is nice to use tuples rather than lists.